docs/docs/hydro/locations/processes.md

```diff
@@ -3,8 +3,35 @@ sidebar_position: 0
 ---
 
 # Processes
-:::caution
+The simplest type of location in Hydro is a process. A process represents a single machine running a piece of a Hydro program. When creating a process, you can pass in a type parameter that will be used as a marker to distinguish that process from others (and will also be used to mark logs originating at that process). For example, you can create a process with a marker of `Leader` to represent a leader in a distributed system:
```

Comment:
Maybe mention that each Hydro process is bound to its own operating system process? Seems important for understanding the sequentiality and security model for a Hydro process. Obviously this *could* change in future, but probably good to doc what's there now.